### DISCUSSION TITLE
Consider adding Orca to SeaORM showcase!

### DISCUSSION THREAD

**Discussion by billy1624** [orcaci/orca#15] [Ideas]
Hey @itsparser, feel free to submit a PR to showcase your project!
- https://github.com/SeaQL/sea-orm/issues/403

**Comment by 0xvasanth** [comment]
@billy1624  can you merge this PR -> [link](https://github.com/SeaQL/sea-orm/pull/714/files)

**Comment by billy1624** [comment]
Thanks! @itsparser
